packages/css/src/components/badge/badge.template.ts

```diff
@@ -7,7 +7,8 @@ const statusMap = new Map<string, string>([
   ['success', 'Gelukt'],
   ['info', 'Opmerking'],
   ['warning', 'Waarschuwing'],
-  ['danger', 'Fout']
+  ['danger', 'Fout'],
+  ['outline', 'Rand']
```

Comment:
Het slaat natuurlijk nergens op dat de sr-only van deze badge `Rand:` wordt. Gelukkig hebben we vanmiddag vastgesteld dat we die sr-only tekstjes er helemaal uit gaan gooien, omdat badges net zo vaak 'betekenisloos' worden ingezet ('gewoon, om de kleur') als dat ze een betekenisvolle tekst verdienen. En dat als er een betekenisvolle tekst is, die net zo goed ook echt leesbaar zou kunnen zijn. En als er toch een sr-only tekst is, dat dat prima mag, maar dat het geen verplichting meer is.